src/backoffice/documents/dashboards/redirect-management/dashboard-redirect-management.element.ts

```diff
@@ -1,17 +1,320 @@
+									Search<uui-icon name="umb:search"></uui-icon>
+								</uui-button>
+							</div>
+							<uui-button
+								label="Disable URL tracker"
+								look="outline"
+								color="danger"
+								@click="${this._disableRedirectHandler}">
+								Disable URL tracker
+							</uui-button> `
+					: html`<uui-button
+							label="Enable URL tracker"
+							look="primary"
+							color="positive"
+							@click="${this._toggleRedirect}">
+							Enable URL tracker
+					  </uui-button>`}
+			</div>
+
+			${this._total && this._total > 0
+				? html`<div class="${this._trackerStatus ? 'trackerEnabled' : 'trackerDisabled'}">${this.renderTable()}</div>`
+				: this._filter?.length
+				? this._renderZeroResults()
+				: this.renderNoRedirects()} `;
+	}
+
+	private _renderZeroResults() {
+		return html`<uui-box>
+			<strong>No redirects matching this search criteria</strong>
+			<p>Double check your search for any error or spelling mistakes.</p>
+		</uui-box>`;
+	}
+
+	private renderNoRedirects() {
+		return html`<uui-box>
+			<strong>No redirects have been made</strong>
+			<p>When a published page gets renamed or moved, a redirect will automatically be made to the new page.</p>
+		</uui-box>`;
+	}
+
+	private renderTable() {
+		return html`<uui-box style="--uui-box-default-padding: 0;">
+				<uui-table>
+					<uui-table-head>
+						<uui-table-head-cell style="width:10%;">Culture</uui-table-head-cell>
+						<uui-table-head-cell>Original URL</uui-table-head-cell>
+						<uui-table-head-cell style="width:10%;"></uui-table-head-cell>
+						<uui-table-head-cell>Redirected To</uui-table-head-cell>
+						<uui-table-head-cell style="width:10%;">Actions</uui-table-head-cell>
+					</uui-table-head>
+					${this._redirectData?.map((data) => {
+						return html` <uui-table-row id="redirect-key-${data.key}">
+							<uui-table-cell> ${data.culture || '*'} </uui-table-cell>
+							<uui-table-cell>
+								<a href="${data.originalUrl || '#'}" target="_blank"> ${data.originalUrl}</a>
+								<uui-icon name="umb:out"></uui-icon>
+							</uui-table-cell>
+							<uui-table-cell>
+								<uui-icon name="umb:arrow-right"></uui-icon>
+							</uui-table-cell>
+							<uui-table-cell>
+								<a href="${data.destinationUrl || '#'}" target="_blank"> ${data.destinationUrl}</a>
+								<uui-icon name="umb:out"></uui-icon>
+							</uui-table-cell>
+							<uui-table-cell>
+								<uui-action-bar style="justify-self: left;">
+									<uui-button
+										label="Delete"
+										look="secondary"
+										.disabled=${!this._trackerStatus}
+										@click="${() => this._removeRedirectHandler(data)}">
+										<uui-icon name="delete"></uui-icon>
+									</uui-button>
+								</uui-action-bar>
+							</uui-table-cell>
+						</uui-table-row>`;
+					})}
+				</uui-table>
 			</uui-box>
-		`;
+			${this._renderPagination()}
+			</uui-scroll-container
+		>`;
+	}
+
+	private _renderPagination() {
+		if (this._total) {
```

Comment:
Consider if it makes sense to render the pagination at all if there is only 1 page.